libs/.eslintrc

```diff
@@ -13,6 +13,7 @@
     "@typescript-eslint/explicit-function-return-type": "off",
     "@typescript-eslint/restrict-template-expressions": "off",
     "@typescript-eslint/strict-boolean-expressions": "off",
+    "@typescript-eslint/naming-convention": "off",
```

Comment:
👀 what's this disabled for?

Reply:
yeah weirdly it was complaining about `const blah_blah` and `const {blah_blah}`, not really sure why it just started happening though so can double check on it

Reply:
Might be worth disabling it in cases where we need to override instead of all up

Reply:
it was really weird to configure so just disabled all, but id like to come back to this, especially for "eslint-config-audius"

Reply:
huh dang. I think it's probably worth trying to keep on to help prevent accidental introduction of new snake_case names if possible! I imagine if we get rid of the `parseApiHealthStatusReason` we could probably get leave this on without errors?